graphics/VIBES/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake  1.1
+
+github.setup        ENSTABretagneRobotics VIBES 0.2.3
+categories          graphics
+platforms           darwin
+maintainers         nomaintainer
+
+description         Visualizer for Intervals and Boxes
+long_description    VIBes is a visualization system that aims at providing\
+                    people working with interval methods a way to display results (boxes, pavings),\
+                    without worrying with GUI programming. It provides drawing functions accessible\
+                    from a lot of programming languages, without complex installation and library\
+                    dependencies. The main design goal of VIBes is to be cross-platform, available\
+                    from different programming languages, simple to set-up, easy to port to a new\
+                    language.
+
+checksums           rmd160  165c8e17970ba55ed78952b674286099cbfd1c68 \
+                    sha256  ff554b0ea18f8c41190894944f614552925ee1c7d94e4133f7e42c0eb864dcda \
+                    size    669976
+
+depends_build       port:qt5\
```

Comment:
If this port uses qt5, then it should be using the qt5 portgroup to build.

Reply:
Also, use `depends_build-append` instead to not overwrite portgroup dependencies.